onnxruntime/core/providers/cpu/tensor/split.cc

```diff
@@ -28,6 +30,10 @@ Status Split::Compute(OpKernelContext* context) const {
 
   if (data_type == DataTypeImpl::GetType<float>())
     status = ComputeImpl<float>(*context, input);
+  else if (data_type == DataTypeImpl::GetType<int32_t>())
```

Comment:
I think it would be better even if such changes (however trivial) are accompanied by relevant tests to make sure coverage standards are upheld :)

Reply:
added.